frontend/package.json

```diff
@@ -8,8 +8,7 @@
     "build": "tsc && vite build",
     "lint": "biome check --write --staged --no-errors-on-unmatched --files-ignore-unknown=true ./",
     "preview": "vite preview",
-    "test": "jest",
-    "testEnvironment": "jsdom"
+    "test": "jest"
```

Comment:
we need jsdom to test react code

Reply:
![image](https://github.com/user-attachments/assets/8107d80d-d712-4ffb-9e10-55ce1fcd23ee)
![image](https://github.com/user-attachments/assets/3460760b-94f7-4c20-bf23-5482a029cdcc)

Reply:
ReferenceError: TextEncoder is not defined error occurs if switched to jsdom